lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -750,6 +763,91 @@ def get_valid_flags_by_version(cursor):
     ]
 
 
+class PgMembership():
+    def __init__(self, module, cursor, target_roles, groups, fail_on_role=True):
```

Comment:
Is there a reason the last optional option is called the same as a module option, but seems to be unrelated to it?

Reply:
(Actually, this constructor argument is never set.)

Reply:
the same reason as in the comment above.